test/pino-roll.test.js

```diff
@@ -243,3 +243,115 @@ test('symlink rotates on roll', async ({ equal, ok, resolves }) => {
   const content = await readFile(linkPath, 'utf8')
   ok(content.includes('#4'), 'symlink contains fourth log')
 })
+
+test('rotate file with date format based on frequency', async ({ ok, rejects }) => {
```

Comment:
Can you move those to a different test file? This is becoming too long.

Reply:
@mcollina Done!